hibernate-core/src/test/java/org/hibernate/orm/test/jpa/criteria/basic/BasicCriteriaUsageTest.java

```diff
@@ -63,31 +69,6 @@ public void testTrivialCompilation() {
 		em.close();
 	}
 
-	@Test
-	@TestForIssue(jiraKey = "HHH-8283")
-	public void testDateCompositeCustomType() {
```

Comment:
Why did you remove the test?

Reply:
The test was removed because Steve has changed the entity class field annotation so it doesn't apply any more. I revert back this testing case so we won't lose one single testing case. I need to confirm with Steve the rationale behind the entity class changing (maybe to make his testing cases pass in his PR and now we might be able to revert back).